ss58-registry-derive/Cargo.toml

```diff
@@ -0,0 +1,19 @@
+[package]
+name = "ss58-registry-derive"
```

Comment:
This is not a derive macro.

And do we really need an extra proc macro for this? Why isn't that a build.rs?

Reply:
I don't think there's much difference between a proc macro and a build.rs. They both require a rustc invocation and then are executed. I think it's more honest to do this as a proc macro than to try and shoehorn it into a build.rs. I agree it's not a custom derive so I should call it something else.

Reply:
This also means one less crate that pollutes crates.io for no reason. A lot of projects are doing this, e.g. prost. I would also favor this approach.